.github/workflows/comment-on-removed-files.yml

```diff
@@ -0,0 +1,64 @@
+name: Comment on removed .md files
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  detect-removed-markdown:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    
+    steps:
+      - uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const prNumber = context.payload.pull_request.number;
+
+            // Get all files in the PR
+            const { data: files } = await github.rest.pulls.listFiles({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: prNumber
+            });
+
+            // Filter for removed .md files
+            const removedMd = files
+              .filter(f => f.status === 'removed' && f.filename.endsWith('.md'))
```

Comment:
We might want to warn on both `removed` AND `renamed` since the URL is determined by the file path.

Reply:
Good idea 👍